xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -564,7 +794,10 @@ void CPVRTimerInfoTag::GetNotificationText(std::string &strText) const
       strText = StringUtils::Format("%s: '%s'", g_localizeStrings.Get(19224).c_str(), m_strTitle.c_str());
     break;
   case PVR_TIMER_STATE_SCHEDULED:
-    strText = StringUtils::Format("%s: '%s'", g_localizeStrings.Get(19225).c_str(), m_strTitle.c_str());
+    if (IsRepeating())
+      strText = StringUtils::Format("%s: '%s'", g_localizeStrings.Get(826).c_str(), m_strTitle.c_str());
```

Comment:
Good point ... and I think you found the only place where I have not done it. Will extend new and existing code in this method.